src/main/resources/templates/email/feedbackToInterviewee.txt

```diff
@@ -0,0 +1,26 @@
+Thank you for practicing your interviewing skills using InterviewMe on {{formatted_date}}. Your interviewer has submitted feedback for your interview which you can view below. 
+
+The values 1-10 represent whether your interviewer strongly disagreed(1) or agreed(10) with the prompt. 
+
+The interviewee communicated their thought process as they went along: {{question_1}}
```

Comment:
Optional: it might end up making the code that fills in this template simpler if you started the numbering from zero. However, I'm not confident about that, so feel free to leave as-is.

Reply:
Will leave as-is for now, and will change if issues occur.